spec/create_table_statement_spec.cr

```diff
@@ -24,4 +24,31 @@ describe LuckyMigrator::CreateTableStatement do
       email text)
     SQL
   end
+
+  it "can create tables with indexes" do
+    statement = LuckyMigrator::CreateTableStatement.new(:users).build do
+      add name : String, index: true
+      add email : String, index: true, unique: true
+    end
+
+    statement.should eq <<-SQL
+    CREATE TABLE users (
+      id serial PRIMARY KEY,
+      created_at timestamp NOT NULL,
+      updated_at timestamp NOT NULL,
+      name text NOT NULL,
+      email text NOT NULL)
+    CREATE INDEX users_name_index ON users USING btree (name);
+    CREATE UNIQUE INDEX users_email_index ON users USING btree (email);
```

Comment:
A little thing, but could you add two spaces to these so it lines up nicely?

Reply:
Sure thing!